examples/gallery/scripts/serve.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
```

Comment:
This app is intended just for dev use? Will this be hosted anywhere? Seems a bit bare-bones (e.g. no security considerations, no logging/analytics, etc) for prod deployment.

Reply:
`npm run build` will write index page and each example to individual html files.